server-common/src/test/java/org/apache/kafka/server/util/timer/TimerTest.java

```diff
@@ -77,7 +77,7 @@ public void setup() {
     @AfterEach
     public void teardown() throws Exception {
         timer.close();
-        TestUtils.waitForCondition(() -> timer.isExecutorTerminated(), "timer excutor not terminated");
+        TestUtils.waitForCondition(() -> timer.isTerminated(), "timer executor not terminated");
```

Comment:
`timer::isTerminated`

Reply:
God... I definitly need to open the IDE check for this. Many thanks